ddtrace/span.py

```diff
@@ -457,6 +458,13 @@ def _remove_exc_info(self):
     def pprint(self):
         # type: () -> str
         """ Return a human readable version of the span. """
+
+        def _dict_section(name, data):
+            # type: (str, Union[Dict[str, Any], Dict[Union[bytes, str], Any]]) -> List[Tuple[str, str]]
```

Comment:
This type hint looks a bit funny. @majorgreys Do you reckon the type of `self.metrics` could be changed to `_MetaDictType`?